py/desispec/pipeline/tasks/psf.py

```diff
@@ -100,7 +100,7 @@ def _run_defaults(self):
         # if that directory does not exist, use a default NERSC
         # location.
         opts["lamplines"] = \
-            "/project/projectdirs/desi/software/edison/specex/specex-0.3.9/data/specex_linelist_desi.txt"
+            "/global/common/software/desi/cori/desiconda/current/code/specex/master/data/specex_linelist_desi.txt"
```

Comment:
For reproducibility, let's tie this to a specific version and not master, e.g. "/global/common/software/desi/cori/desiconda/20180709-1.2.6-spec/code/specex/0.6.2/data/specex_linelist_desi.txt".  Or maybe
`os.path.expandvars('$SPECEX/data/specex_linelist_desi.txt')` and then raise and exception if that doesn't exist.

Would be better if specex itself had a way to query where the linelist file is for whatever version is installed / configured, e.g.
```
import pkg_resources
pkg_resources.resource_filename('specex', 'data/specex_linelist_desi.txt')
```
I haven't checked how the pipeline wrapper finds specex, but perhaps that could be leveraged instead to avoid special environment variables or hardcoding a path (and then forgetting to update it when we move to a new desiconda version).